CHANGES.md

```diff
@@ -13,6 +13,33 @@ development source code and as such may not be routinely kept up to date.
 
 # __NEXT__
 
+## Improvements
+
+* Use of an alternate Conda package repository is now supported during
+  `nextstrain setup conda` and `nextstrain update conda` if you cannot or do
+  not want to use the default package repository hosted by Anaconda.  Set the
+  [`NEXTSTRAIN_CONDA_CHANNEL_ALIAS`][] environment variable to the base URL of
+  the repository.  This corresponds to the [`channel_alias` Conda config
+  setting][].
+  ([#436](https://github.com/nextstrain/cli/pull/436))
+
+* The Conda runtime no longer requires Rosetta 2 for macOS running on aarch64
+  (aka arm64, Apple Silicon, M1/M2) hardware.  This improves performance when
```

Comment:
Suggestion: hint that the list is non-exhaustive. Some users may already be on M4!

```suggestion
  (aka arm64, Apple Silicon, M1/M2/…) hardware.  This improves performance when
```

Reply:
Good call!

Reply:
Added in [re-push](https://github.com/nextstrain/cli/compare/9075b98fecf897817bb53e5bfdbbdc712751d5c4..9cd6964671e774f2b4d6e7118af4c00b5afd21fe).